flag_engine/features/models.py

```diff
@@ -19,41 +21,75 @@ def __hash__(self):
         return hash(self.id)
 
 
-@dataclass
-class MultivariateFeatureOptionModel:
+class MultivariateFeatureOptionModel(BaseModel):
     value: typing.Any
     id: int = None
 
 
-@dataclass
-class MultivariateFeatureStateValueModel:
+class MultivariateFeatureStateValueModel(BaseModel):
     multivariate_feature_option: MultivariateFeatureOptionModel
-    percentage_allocation: float
+    percentage_allocation: confloat(ge=0, le=100)
     id: int = None
```

Comment:
Should we use `conint` here to make sure it is a positive integer?

Reply:
Why, though? Current schemas don't support that constraint, do they?

Reply:
I can think of a couple — minimize the diff, try to reach 1:1 feature parity with the current engine version.
We have a typing PR #168 where we can address this.

Reply:
Ok.